### PR TITLE
Metadata fixes

### PR DESCRIPTION
**Pull request by Mathieu-Be**
Fixing various metadata issues. Assets weren't uploaded in the same order as it is set in the `Brotherhood` enum, but since this is the thing that impacts images, I need to update the enum and all associated arrays.

[Notion page](https://www.notion.so/traderjoexyz/Smol-Joes-contract-upgrade-73b26c772f6248c9aba3b7ab2306e926)

### CODE REVIEW

File: src/interfaces/ISmolJoeArt.sol [hunk 1]
```diff
@@ -43,12 +43,12 @@ interface ISmolJoeArt {
...
         Creatives,
         Gentlemans,
+        Superheros,
         MagicalBeings,
-        Military,
         Musicians,
         Outlaws,
-        Religious,
-        Superheros
+        Military,
```
Comment (0x0Louis) [inline src/interfaces/ISmolJoeArt.sol:50]: Shouldn't this be renamed `Warrior` ?

File: src/interfaces/ISmolJoeArt.sol [hunk 2]
```diff
@@ -43,12 +43,12 @@ interface ISmolJoeArt {
         Athletes,
         Creatives,
         Gentlemans,
+        Superheros,
```
Comment (0x0Louis) [inline src/interfaces/ISmolJoeArt.sol:46]: Shouldn't this be renamed `Heroes` ?

File: src/interfaces/ISmolJoeArt.sol [hunk 3]
```diff
@@ -43,12 +43,12 @@ interface ISmolJoeArt {
...
         MagicalBeings,
-        Military,
         Musicians,
         Outlaws,
-        Religious,
-        Superheros
+        Military,
+        Religious
```
Comment (0x0Louis) [inline src/interfaces/ISmolJoeArt.sol:51]: Shouldn't this be renamed `Worshipers` ?

**Review by 0x0Louis — APPROVED** [review]

File: src/interfaces/ISmolJoeArt.sol (hunk 1 above)
Comment (Mathieu-Be) [reply src/interfaces/ISmolJoeArt.sol]: Yeah I guess, for consistency

**Review by 0x0Louis — APPROVED** [review]

**Pull request merged**